libV2/validationUtils.js

```diff
@@ -2371,7 +2371,7 @@ function checkResponses (context, transaction, transactionPathPrefix, schemaPath
   // for each response, find the appropriate response from schemaPath, and then validate response body and headers
   async.map(responses, (response, responseCallback) => {
     let thisResponseCode = _.toString(response.code),
-      thisSchemaResponse = _.get(schemaPath, ['responses', thisResponseCode]),
+      thisSchemaResponse = _.get(schemaPath, ['responses', thisResponseCode], _.get(schemaPath, 'responses.default')),
```

Comment:
@barshan23 Could we add a follow-up to add tests for this case, so there are no regressions?